BUILD_STATUS.md

```diff
@@ -0,0 +1,22 @@
+# Build Status
+
+| Workflow  | Main | 23.lts.1+ | 22.lts.1+ | 21.lts.1+ | 20.lts.1+ | 19.lts.1+ | RC11 | COBALT 9 |
+| --------- | ---- | --------- | --------- | --------- | --------- | --------- | ---- | ---------|
+| Lint      | [![lint](https://github.com/youtube/cobalt_sandbox/actions/workflows/lint.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt_sandbox/actions/workflows/lint.yaml) | | | | | | | |
```

Comment:
Can you permalink to workflows filtered in the same way we filter events for the badge?

Reply:
I believe it is, so let's link to the filtered list

Reply:
Done